tidb_cloud/metadata.csv

```diff
@@ -4,11 +4,9 @@ tidb_cloud.db_failed_queries_total,count,,query,,Total number of execution error
 tidb_cloud.db_connections,gauge,,connection,,Current number of connections in TiDB server,0,tidb_cloud,
 tidb_cloud.db_query_duration_seconds.count,count,,query,,Total number of handled queries in server,0,tidb_cloud,
 tidb_cloud.db_query_duration_seconds.sum,count,,second,,Sum of handled query duration in server,0,tidb_cloud,
-tidb_cloud.storage_used_bytes,gauge,,byte,,Disk usage bytes of TiKV/TiFlash instances,0,tidb_cloud,
-tidb_cloud.storage_capacity_bytes,gauge,,byte,,Disk capacity bytes of TiKV/TiFlash instances,0,tidb_cloud,
-tidb_cloud.storage_tikv_io_bytes_total,count,,byte,,IO read/write bytes of TiKV instances,0,tidb_cloud,
-tidb_cloud.process_cpu_seconds_total,count,,second,,Sum of CPU usage seconds of TiDB/TiKV/TiFlash instances,0,tidb_cloud,
-tidb_cloud.process_cpu_used_cores,gauge,,core,,CPU usage cores of TiDB/TiKV/TiFlash instances,0,tidb_cloud,
+tidb_cloud.node_storage_used_bytes,gauge,,byte,,Disk usage bytes of TiKV/TiFlash instances,0,tidb_cloud,
+tidb_cloud.node_storage_capacity_bytes,gauge,,byte,,Disk capacity bytes of TiKV/TiFlash instances,0,tidb_cloud,
+tidb_cloud.node_cpu_seconds_total,count,,second,,Sum of CPU usage seconds of TiDB/TiKV/TiFlash instances,0,tidb_cloud,
 tidb_cloud.node_cpu_capacity_cores,gauge,,core,,CPU limit cores of TiDB/TiKV/TiFlash instances,0,tidb_cloud,
-tidb_cloud.process_memory_used_bytes,gauge,,byte,,Resident memory bytes of TiDB/TiKV/TiFlash instances,0,tidb_cloud,
+tidb_cloud.node_memory_used_bytes,gauge,,byte,,Resident memory bytes of TiDB/TiKV/TiFlash instances,0,tidb_cloud,
```

Comment:
Is there any chance that users may be using an older version of tidb cloud that would emit the old versions of these metrics? If so, we should keep the old metric names here too. Otherwise, we can keep the change.

Reply:
Users will not use older versions of metrics.

The metrics generated in the TiDB Cloud are now the newer versions already.